xbmc/guilib/GUITextBox.cpp

```diff
@@ -394,7 +394,10 @@ unsigned int CGUITextBox::GetRows() const
 
 int CGUITextBox::GetNumPages() const
 {
-  return (GetRows() + m_itemsPerPage - 1) / m_itemsPerPage;
+  if (m_itemsPerPage == 0)
```

Comment:
You can make this a bit more concise using a ternary operator:

```
int CGUITextBox::GetNumPages() const
{
  return m_itemsPerPage > 0 ? (GetRows() + m_itemsPerPage - 1) / m_itemsPerPage : 0;
}
```

Reply:
Cool, more concise